tests/unit/workflows/test_workflows_tasks_refextract.py

```diff
@@ -24,15 +24,16 @@
 
 import os
 import pkg_resources
+from mocks import MockEng, MockObj
 
 from inspire_schemas.utils import load_schema
+
 from inspirehep.dojson.utils import validate
 from inspirehep.modules.workflows.tasks.refextract import (
     extract_journal_info,
     extract_references,
 )
-
-from mocks import MockEng, MockObj
+from inspirehep.utils.record import get_value
```

Comment:
As for the imports, the only thing that should change here is that you should add the last one, and everything else was ok before.